src/openrct2/object/ObjectRepository.cpp

```diff
@@ -98,13 +98,12 @@ class ObjectRepository final : public IObjectRepository
     std::vector<ObjectRepositoryItem>   _items;
     QueryDirectoryResult                _queryDirectoryResult = { 0 };
     ObjectEntryMap                      _itemMap;
-    uint16                              _languageId = 0;
-    sint32                                 _numConflicts;
+    uint16                              _languageId   = 0;
+    sint32                              _numConflicts = 0;
 
 public:
-    ObjectRepository(IPlatformEnvironment * env)
+    ObjectRepository(IPlatformEnvironment * env) : _env(env)
     {
-        _env = env;
     }
```

Comment:
@IntelOrca I was misunderstood there then. I disliked the
```
, value(0)
```
over
```
value(0),
```
rather than initialization list vs assignement in the constructor's body.